packages/appium/test/helpers.js

```diff
@@ -0,0 +1,35 @@
+import path from 'path';
+import {insertAppiumPrefixes} from '../lib/utils';
+import findUp from 'find-up';
+import getPort from 'get-port';
+
+const TEST_HOST = 'localhost';
+
+// monorepo root.  this cannot be hardcoded because:
+// 1. we may be in 'build/test' or 'test', depending on our config
+// 2. Node.js does not support `__dirname` in an ESM context & Babel doesn't fake it well
+const PROJECT_ROOT = path.dirname(findUp.sync('.git', {type: 'directory'}));
+const TEST_FAKE_APP = path.join(PROJECT_ROOT, 'packages', 'fake-driver', 'test', 'fixtures', 'app.xml');
+
+const BASE_CAPS = {
+  automationName: 'Fake',
+  platformName: 'Fake',
+  deviceName: 'Fake',
+  app: TEST_FAKE_APP
+};
+const W3C_PREFIXED_CAPS = {...insertAppiumPrefixes(BASE_CAPS)};
+const W3C_CAPS = {
+  alwaysMatch: {...W3C_PREFIXED_CAPS},
+  firstMatch: [{}],
+};
+
+let TEST_PORT;
+/**
+ * Returns a free port; one per process
+ * @returns {Promise<number>} a free port
```

Comment:
we skip `Promise` and only put the actual value type, since an IDE knows anyway that the result of an `async` function is always a promise

Reply:
that's true that the IDE knows, but if we're being explicit about `return await` it would be consistent to be explicit about this, yes?

Reply:
I never use Promise notation anywhere for async returns. Yolo :)

Reply:
we should just port everything to typescript and be done with it :trollface:

Reply:
@jlipps imo it's not needed.  properly-written docstrings are easily understood by typescript, and you can just use it as a type linter (and you get that info in the IDE!).   I wrote [report-toolkit](https://github.com/ibm/report-toolkit) this way.